src/csharp/Grpc.IntegrationTesting/SslCredentialsTest.cs

```diff
@@ -44,17 +44,23 @@ public class SslCredentialsTest
 
         string rootCert;
         KeyCertificatePair keyCertPair;
+        string certChain;
+        List<ChannelOption> options;
+        bool isHostEqual;
+        bool isPemEqual;
 
         public void InitClientAndServer(bool clientAddKeyCertPair,
                 SslClientCertificateRequestType clientCertRequestType)
         {
             rootCert = File.ReadAllText(TestCredentials.ClientCertAuthorityPath);
+            certChain = File.ReadAllText(TestCredentials.ServerCertChainPath);
+            certChain = certChain.Replace("\r", string.Empty);
```

Comment:
why this change?

Reply:
The `certChain` obtained from the file contains `\r`, but in the callback, the string passed by the unmanaged code doesn't contain `\r`, which lead the equality line 246 to fail. I guess I could strip the `\r` in the callback, let me know if you think it is better.